### PR TITLE
Enable Cube Orange built in ADS-B receiver by default

### PR DESCRIPTION
**Pull request by Etruscian**
Please use [PX4 Discuss](http://discuss.px4.io/) or [Slack](http://slack.px4.io/) to align on pull requests if necessary. You can then open draft pull requests to get early feedback.

**Describe problem solved by this pull request**
This pull request enables the built in ADS-B receiver of the Cube Pilot Cube Orange

**Test data / coverage**
Tested by multiple people and myself on the bench

**Additional context**
The pull request was created based on a slack discussion.

### CODE REVIEW

File: boards/cubepilot/cubeorange/init/rc.board_mavlink [hunk 1]
```diff
@@ -5,3 +5,6 @@
 
 # Start MAVLink on the USB port
 mavlink start -d /dev/ttyACM0
+
+# Start ADS-B receiver mavlink connection
+mavlink start -d /dev/ttyS4
```
Comment (LorenzMeier) [inline boards/cubepilot/cubeorange/init/rc.board_mavlink:10]: Newline is missing - sorry about the nitpick, but I'm trying to also give feedback for future PRs.

**Review by LorenzMeier — CHANGES_REQUESTED** [review]

File: boards/cubepilot/cubeorange/init/rc.board_mavlink (hunk 1 above)
Comment (Etruscian) [reply boards/cubepilot/cubeorange/init/rc.board_mavlink:10]: Which newline do you mean? And nitpicking is fine if it improves the code/code style ^^

Comment (LorenzMeier) [reply boards/cubepilot/cubeorange/init/rc.board_mavlink:10]: See the red exclamation mark. The line you added does not end in a newline (but should)

Comment (Etruscian) [reply boards/cubepilot/cubeorange/init/rc.board_mavlink:10]: Ah, this was a windows line ending. I didn't see the new line ending. Will change it!

**Comment by dagar** [comment]
If this isn't going to be configurable then please remove this line as well. https://github.com/PX4/PX4-Autopilot/blob/3aa3f542ae79efd217167ef5435175b500a2c1d5/boards/cubepilot/cubeorange/default.cmake#L15

**Comment by Etruscian** [comment]
Good catch @dagar ! Added it

**Review by LorenzMeier — APPROVED** [review]

**Pull request merged**